Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM ghcr.io/celestiaorg/celestia-node:v0.7.2 AS celestia-node
+FROM ghcr.io/celestiaorg/celestia-node:v0.8.1 AS celestia-node
 
 FROM ghcr.io/celestiaorg/celestia-app:v0.12.1
```

Comment:
should be v0.12.2 according to go.mod from v0.8.1 node 
cc: @celestiaorg/devops we need to think about how to maintain this zoo of dockerfiles tbh

Reply:
it still fails

```bash
jcs @ ~/DEV/local-celestia-devnet % docker build . -t celestia-local-devnet-8-1
[+] Building 0.7s (4/4) FINISHED                                                                                                                                       
 => [internal] load build definition from Dockerfile                                                                                                              0.0s
 => => transferring dockerfile: 359B                                                                                                                              0.0s
 => [internal] load .dockerignore                                                                                                                                 0.0s
 => => transferring context: 2B                                                                                                                                   0.0s
 => CANCELED [internal] load metadata for ghcr.io/celestiaorg/celestia-app:v0.12.2                                                                                0.6s
 => ERROR [internal] load metadata for ghcr.io/celestiaorg/celestia-node:v0.8.1                                                                                   0.6s
------
 > [internal] load metadata for ghcr.io/celestiaorg/celestia-node:v0.8.1:
------
ERROR: failed to solve: rpc error: code = Unknown desc = failed to solve with frontend dockerfile.v0: failed to create LLB definition: no match for platform in manifest sha256:1705bb2d5e8a0efa797db6964003c47c7a8cb452a1c92c3f4a9302390cb9f858: not found
```

Reply:
yay. it ran on 0.8.2